pkgs/dartpad_ui/lib/main.dart

```diff
@@ -631,7 +631,10 @@ class DartPadAppBar extends StatelessWidget implements PreferredSizeWidget {
                 if (genAiEnabled) ...[
                   const SizedBox(width: denseSpacing),
                   GeminiMenu(
-                    generateNewCode: () => _generateNewCode(context),
+                    generateNewDartCode:
+                        () => _generateNewCode(context, AppType.dart),
+                    generateNewFlutterCode:
+                        () => _generateNewCode(context, AppType.flutter),
                     updateExistingCode: () => _updateExistingCode(context),
```

Comment:
A more "Fluttery" API may be to pass a list of `MenuItemButton`s that know what to do, instead of the callbacks, but since this is not going to have many more options ever this is probably fine!